step.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# download the Bitrise Build Cache CLI
+curl -sSfL 'https://raw.githubusercontent.com/bitrise-io/bitrise-build-cache-cli/main/install/installer.sh' | sh -s -- -b /tmp/bin -d
```

Comment:
I think you should also add the retry flags to this

Reply:
Another concern: this always installs the latest version, is this okay for the usecase? It certainly goes against pinning step versions in order to pin the step behavior, so maybe a dedicated step input for the CLI tool version would be better.